pkg/models/app.go

```diff
@@ -10,4 +12,20 @@ type App struct {
 	NumOfCurrentInstalls  *int       `json:"numOfCurrentInstalls,omitempty"`
 	NumOfAppLaunches      *int       `json:"numOfAppLaunches,omitempty"`
 	DeployedVersions      *[]Version `json:"deployedVersions,omitempty"`
+	DeletedAt             string     `json:"deletedAt,omitempty"`
+}
+
+func NewAppByDevice(sdkInfo *Device) *App {
+	app := new(App)
```

Comment:
See that the new(Type) is required to initialize a new instance of the object.

Reply:
It is not actually required to use the `new` function:

See https://github.com/aerogear/mobile-security-service/blob/8c837ed3d8fde60b55e2aa26f429d5ef11cb4d86/pkg/models/device.go#L23

Reply:
Yes, you are correct. The &Type{} will do the same, however, it is not a good practice when has many values. The new(Type) help keep the code clean and more understandable.